src/main/scala/com/mozilla/telemetry/stats/MannWhitneyU.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.mozilla.telemetry.stats
+
+
+// Constants
+
+object MannWhitneyU {
+
+  def erf(x: Double): Double =  {
+    // Constants.
+    val a1: Double = 0.254829592
+    val a2: Double = -0.284496736
+    val a3: Double = 1.421413741
+    val a4: Double = -1.453152027
+    val a5: Double = 1.061405429
+    val p: Double = 0.3275911
+
+    if (x == 0) {
+      return 0.0
```

Comment:
If `x == 0`, this result is `9.999999717180685E-10`; I think we can safely remove the early return.